cmd/uplink/cmd/mount.go

```diff
@@ -0,0 +1,235 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build linux darwin netbsd freebsd openbsd
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"sync"
+
+	"github.com/hanwen/go-fuse/fuse"
+	"github.com/hanwen/go-fuse/fuse/nodefs"
+	"github.com/hanwen/go-fuse/fuse/pathfs"
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/pkg/paths"
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/pkg/ranger"
+	"storj.io/storj/pkg/storage/meta"
+	"storj.io/storj/pkg/storage/objects"
+	"storj.io/storj/pkg/utils"
+	"storj.io/storj/storage"
+)
+
+func init() {
+	addCmd(&cobra.Command{
+		Use:   "mount",
+		Short: "Mount a bucket",
+		RunE:  mountBucket,
+	})
+}
+
+func mountBucket(cmd *cobra.Command, args []string) (err error) {
+	if len(args) == 0 {
+		return fmt.Errorf("No bucket specified for mounting")
+	}
+	if len(args) == 1 {
+		return fmt.Errorf("No destination specified")
+	}
+
+	ctx := process.Ctx(cmd)
+
+	bs, err := cfg.BucketStore(ctx)
+	if err != nil {
+		return err
+	}
+
+	u, err := utils.ParseURL(args[0])
```

Comment:
Use the new FPath instead. See how the rest of the commands were converted in #423.